ServerRecording/RecordingApi.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 	<PropertyGroup>
-		<TargetFramework>net5.0</TargetFramework>
+		<TargetFramework>net6.0</TargetFramework>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="Azure.Communication.CallingServer" Version="1.0.0-beta.3" />
-		<PackageReference Include="Azure.Communication.Identity" Version="1.0.1" />
-		<PackageReference Include="Azure.Messaging.EventGrid" Version="4.6.0" />
-		<PackageReference Include="Azure.Storage.Blobs" Version="12.9.1" />
-		<PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="3.1.18" />
-		<PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="5.0.9" />
+		<PackageReference Include="Azure.Communication.CallingServer" Version="1.0.0-alpha.20220919.1" />
```

Comment:
Can you please update package reference to Azure.Communication.CallAutomation?  Package has been renamed.